config/ro2018-czu-waypoints.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "init": {
+            "maxspeed": 10,
+            "waypoints": [[50.1283538, 14.3752654],
+                          [50.1281866, 14.3736720]]
```

Comment:
Just a thought. Here we decide in the config file which module should get the data. Maybe it could work the other way around. We could have a place for a general config that could be shared among all modules. Each module could peak inside and use whatever it finds useful.

Reply:
OK, it would be like global variables ... so maybe. In this particular case I prefer to give waypoints to the only interested module and it gets only on `config` on input. But I would keep this idea for the future.